web/templatetags/membership_tags.py

```
@@ -0,0 +1,27 @@
+
+
+@register.filter
+def format_price(value, period=None):
+    """Format price based on billing period."""
+    if value is None:
+        return "$0"
+    return f"${value}"
+
+
+@register.simple_tag
+def is_active_membership(user):
```

Ruff (0.8.2)

Missing return type annotation for public function `is_active_membership`

(ANN201)